test/utils.test.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+/* global describe, it */
+const assert = require("assert").strict;
+
+const request = require('supertest');
+const app = require('../app');
+const { connectMongoDB, getRecordFromDB, capitalizeName } = require('../lib/utils');
+const countrySchema = require('../models/country_schema');
+
+describe('API requests on app', function() {
+  this.timeout(10000);
+
+  before(async function() {
+    // set up database
+    await connectMongoDB();
+  });
+
+  it('should return an expected GET /api/ response', function() {
+    return request(app)
+      .get('/')
+      .expect(200)
+  });
+});
+
+//Test getRecordFromDB()
+describe('Test getRecordFromDB()', function() {
+  this.timeout(10000);
+  it('should return a country nane from DB', async() => {
```

Comment:
typo; should be "name"

Reply:
"should return related data of the requested country"

Reply:
Got it!